### PR TITLE
Add Bengali channels

### PR DESCRIPTION
**Pull request by freearhey**
- added more Bengali channels
- updated channels description

### CODE REVIEW

**Review by dct-infra — DISMISSED** [review]
Terrific job! By the way thanks!

File: channels/bd.m3u [hunk 1]
```diff
@@ -1,9 +1,61 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AnandTV.bd" tvg-country="BD" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/0IuoIXI.png" group-title="",Anand TV (720p)
-https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
-#EXTINF:-1 tvg-id="ChannelT1.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/dyoXd9c.png" group-title="",Channel T1 [Not 24/7]
+#EXTINF:-1 tvg-id="ADNBanglaUK.bd" tvg-country="UK" tvg-language="Bengali" tvg-logo="http://atnbanglauk.tv/wp-content/uploads/2015/04/sized1.png" group-title="General",ADN Bangla UK (576p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbanglauk-off.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u:3]: Cannot connect. 

HTTP error 403 Forbidden

File: channels/bd.m3u [hunk 2]
```diff
@@ -1,9 +1,61 @@
...
+#EXTINF:-1 tvg-id="ATNBangla.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/ATN_Bangla.svg/207px-ATN_Bangla.svg.png" group-title="General",ATN Bangla (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbd-8-org.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u:5]: Cannot connect.

HTTP error 403 Forbidden

File: channels/bd.m3u [hunk 3]
```diff
@@ -1,9 +1,61 @@
...
+#EXTINF:-1 tvg-id="ATNNews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/2F1BaT2.gif" group-title="News",ATN News (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnws-sg.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BanglaVision.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/doJuPjL.png" group-title="General",Bangla Vision (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/banglav000.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u:9]: Cannot connect.

HTTP error 403 Forbidden

File: channels/bd.m3u [hunk 4]
```diff
@@ -1,9 +1,61 @@
...
+#EXTINF:-1 tvg-id="BTVChittagong.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/CMUHtv3.png" group-title="Local",BTV Chittagong (720p) [Not 24/7]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvnational-ctg.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BTVWorld.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/1Chamk9.png" group-title="General",BTV World (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvbd-office-sg.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Channel9.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/f/ff/Channel9_bd.svg/350px-Channel9_bd.svg.png" group-title="Local",Channel 9 (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel9hd.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u:19]: Cannot connect.

HTTP error 403 Forbidden

File: channels/bd.m3u [hunk 5]
```diff
@@ -1,9 +1,61 @@
...
+#EXTINF:-1 tvg-id="Channel24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Channel24logo.svg/400px-Channel24logo.svg.png" group-title="News",Channel 24 (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel24-sg-e8e.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ChannelI.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://www.channelionline.com/wp-content/uploads/2019/07/channel-i-logo-1.png" group-title="General",Channel I (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channeli-8-org.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/bd.m3u:23]: Cannot connect.

HTTP error 403 Forbidden

File: channels/bd.m3u (hunk 1 above)
Comment (dct-infra) [reply channels/bd.m3u:3]: 403 on all https://cdn.appv.jagobd.com:444

**Review by dct-infra — COMMENTED** [review]
Might work depending on location

File: channels/ca.m3u [hunk 6]
```diff
@@ -99,6 +101,12 @@ https://d7z3qjdsxbwoq.cloudfront.net/groupa/live/f9809cea-1e07-47cd-a94d-2ddd3e1
 http://159.69.58.154/cpmtv/cpmtv.m3u8
 #EXTINF:-1 tvg-id="CTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/nfcjHAW.png" group-title="",CTV (720p)
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
+#EXTINF:-1 tvg-id="DesheBidesheTV.ca" tvg-country="CA" tvg-language="Bengali" tvg-logo="https://www.deshebideshe.tv/assets/importent_images/footer-logo.png" group-title="General",DesheBideshe TV (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/deshebideshe.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/ca.m3u:105]: Displays banner. No live streaming.

File: channels/bd.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/bd.m3u:3]: There are a bunch of :444 that do stream. only those listed above do not.

**Comment by jkemp814** [comment]
There are a bunch of :444 that do stream. Those listed above do not.

File: channels/bd.m3u (hunk 1 above)
Comment (freearhey) [reply channels/bd.m3u:3]: They all work fine for me, so I guess some of them are just geo-blocked.

File: channels/ca.m3u (hunk 6 above)
Comment (freearhey) [reply channels/ca.m3u:105]: It works fine for me, even through a VPN.
vlcsnap-2021-10-02-22h39m10s936

**Review by jkemp814 — APPROVED** [review]

File: channels/ca.m3u (hunk 6 above)
Comment (jkemp814) [reply channels/ca.m3u:105]: Yes, it's streaming here now. May have been a break, and that was a placeholder.

Comment (freearhey) [reply channels/ca.m3u:105]: Thanks for your help!

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Amazing job Arhey.

**Pull request merged**